apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/IndictmentCount.tsx

```diff
@@ -414,6 +429,76 @@ export const IndictmentCount: React.FC<Props> = (props) => {
           ))}
         </Box>
       )}
+      {indictmentCount.offenses?.includes(
+        IndictmentCountOffense.DrunkDriving,
+      ) && (
+        <Box marginBottom={2}>
+          <InputMask
+            mask={'9,99'}
+            maskPlaceholder={null}
+            value={indictmentCount.substances?.ALCOHOL ?? ''}
+            onChange={(event) => {
+              removeErrorMessageIfValid(
+                ['empty'],
+                event.target.value,
+                bloodAlcoholContentErrorMessage,
+                setBloodAlcoholContentErrorMessage,
+              )
+
+              updateIndictmentCountState(
+                indictmentCount.id,
+                {
+                  substances: {
+                    ...indictmentCount.substances,
+                    ALCOHOL: event.target.value,
+                  },
+                },
+                setWorkingCase,
+              )
+            }}
+            onBlur={(event) => {
+              const value =
+                event.target.value.length > 0
+                  ? `${event.target.value}${'0,00'.slice(
```

Comment:
I don't really understand this, could you please explain a little what this does?

Reply:
I can answer this because Guðjón mentioned it to me yesterday :) It's basically so that the user can write just 1 or 0,2 and we fill out the missing zeroes if needed, so they'd become 1,00 and 0,20 respectively

Reply:
Una explained it well. I will leave this as is.